src/mscorlib/shared/Interop/Unix/System.Native/Interop.GetRandomBytes.cs

```diff
@@ -11,11 +11,11 @@ internal partial class Interop
     internal unsafe partial class Sys
     {
         [DllImport(Interop.Libraries.SystemNative, EntryPoint = "SystemNative_GetNonCryptographicallySecureRandomBytes")]
-        internal static extern unsafe void GetNonCryptographicallySecureRandomBytes(byte* buffer, int length);
+        internal static extern void GetNonCryptographicallySecureRandomBytes(ref byte buffer, int length);
```

Comment:
This change is also introducing slowing things down. There is going to be an extra PInvoke marshaling frame because of the signature is no longer blittable. I think this should stay as `byte*`.

Reply:
https://en.wikipedia.org/wiki/Blittable_types
https://docs.microsoft.com/en-us/cpp/dotnet/calling-native-functions-from-managed-code#performance-considerations

Reply:
Thank's